go-controller/pkg/config/config.go

```diff
@@ -955,6 +997,38 @@ func buildMasterHAConfig(ctx *cli.Context, cli, file *config) error {
 	return nil
 }
 
+func buildHybridOverlayConfig(ctx *cli.Context, cli, file *config) error {
```

Comment:
Has this function been updated (if needed) to take into account Tim's reworking of default values?

Reply:
Fixed.